schema.md

```diff
@@ -0,0 +1,555 @@
+
+### Key `legal/authors-file`
+
+* Type: string (path to file)
+* Presence: optional
+* Example: "doc/AUTHORS.txt"
+
+Some open-source softwares adopt a convention of identify the copyright
+holders through a file that lists all the entities that own the
+copyright. This is common in projects strongly backed by a community
+where there are many external contributors and no clear single/main
+copyright owner. In such cases, this key can be used to refer to the
+authors file, using a path relative to the root of the repository.
+
+### Key `legal/repo-owner`
+
+* Type: string
+* Presence: mandatory
+* Example: "City of Amsterday"
+
+This string describes the entity that owns this repository; this might
+or might not be the same entity who owns the copyright on the code
+itself. For instance, in case of a fork of the original software, the
+`repo-owner` is probably different from the `main-copyright-owner`.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `maintenance`
+
+This section provides information on the maintenance status of the
+software, useful to evaluate whether the software is actively
+developed or not.
+
+### Key `maintenance/type`
+
+* Type: enumerate
+* Presence: mandatory
+* Values: "commercial", "community", "none"
+
+This key describes how the software is currently maintained.
+"Commercial" means that there is a commercial contract that binds
+an entity to the maintenance of the software; "community" means that
+the software is currently maintained by one or more people that donate
+their time to the project; "none" means that the software is not
+actively maintained.
+
+### Key `maintenance/until`
+
+* Type: date (YYYY-MM-DD)
+* Presence: mandatory (if the software is commercially maintained)
+* Example: 2019-07-01
+
+In case of a commercial maintenance, this key must contain the date
+at which the maintenance is going to end.
+
+### Key `maintenance/maintainer`
+
+* Type: string or array of strings
+* Presence: mandatory (if there is a maintenance)
+* Example: "Linus Torvalds", "Italian Linux Group", "Acme Inc."
+
+This key describes the entity or entities that are currently maintaining
+the software. It can contain for instance a company name, an association
+name, a private person. etc.
+
+In case of a commercial agreement (or a chain of such agreements),
+specify the final entities actually contracted to deliver the
+maintenance. Do not specify the software owner unless it is technically
+involved with the maintenance of the product as well.
+
+### Key `maintenance/technical-contacts`
+
+* Type: array of objects
+* Presence: mandatory (if there is a maintenance)
+
+This key describes the physical point of contacts of the technical
+people in charge of the maintenance. The goal is to provide a clear
+contact point for contributors that need to discuss how to best
+implement modifications to the software.
+
+It MUST contain real people names (first name, family name) and direct
```

Comment:
In some countries the default is `(family name, given name)`, and perhaps that is okay as well. I would rephrase as  `real names of individuals`

Reply:
Done